package.json

```diff
@@ -4,6 +4,14 @@
   "description": "HighWatermark Processing with Async Iterators",
   "main": "index.js",
   "types": "index.d.ts",
+  "exports": {
+    ".": "./index.js",
+    "./package.json": "./package.json"
+  },
+  "files": [
+    "index.js",
+    "index.d.ts"
+  ],
```

Comment:
Unrelated change. With `files` set like this, only `LICENSE`, `README.md`, `index.d.ts`, `index.js` and `package.json` will be published to npm. Currently the whole repository is push to the registry (including directories like `.github` or `test`).